### DISCUSSION TITLE
How can I add or remove tables dynamically on same database version

### DISCUSSION THREAD

**Discussion by uribes78** [ujjwalguptaofficial/JsStore#374] [Q&A] [answered]
Hi, as the title indicate, Is it possible to do it?

My goal is to have a dynamic interaction with database, because I'm using the concept of agent that need to work on different campaign, and each campaign have the own records, so when an agent is assigned to a new campaign, I need to create a table in reference with that campaign to store their records.

This is my first time using indexedDb, and the JsStore library is great to achieved my needs. Thank by the way

**Comment by ujjwalguptaofficial** [comment, marked as answer]
Creating dynamic tables is not suited to your needs and its not possbbile. Better to think how to organise your data - 

You can create new campaign and get campaign id and use those campaign id to associate agents and different data.

Even in backend (server side programming) - you need to do these.

**Reply by uribes78** [reply]
Thanks for the answer, actually I solved it using one table for all campaigns with its own id to identify records for each campaign.
